File: Backend/intoTheAbyss/src/network/packets/MoveFloorPacket.java
package network.packets;

public class MoveFloorPacket {
	private String userID;
	private int floor;

	public MoveFloorPacket() {

	}

	public String getUserID() {
		return userID;
	}

	public int getFloorNum() {
		return floor;
	}
}

File: Frontend/IntotheAbyss/app/src/main/java/com/example/intotheabyss/networking/packets/MoveFloorPacket.kt
package com.example.intotheabyss.networking.packets

/**
 * A packet designed to let the interface know when a player has moved from one floor to the next.
 * @constructor Provided the correct parameters, sets the parameters as the payload for the packet
 * @param playerID The player moving between floors.
 * @param floorNum The new floor the player is moving to.
 *
 * @author Benjamin Vogel
 */
class MoveFloorPacket(var playerID: String, var floorNum: Int)